From 6e0638338d5c587d6bbb9c349d1b25382fbdbfce Mon Sep 17 00:00:00 2001
From: Oliver Specht <oliver.specht@cib.de>
Date: Wed, 7 Oct 2015 16:16:20 +0200
Subject: [PATCH] tdf#94804: prefix to save cross reference bookmark fixed

bookmarks as references to numbered paragraphs are now saved
using the prefix '__RefNumPara__' as the import already expects

Change-Id: I6f7c41e95fa9f936a7eb21d4cb7461d4575c492a
Reviewed-on: https://gerrit.libreoffice.org/19229
Tested-by: Jenkins <ci@libreoffice.org>
Reviewed-by: Michael Stahl <mstahl@redhat.com>
(cherry picked from commit 85eb4d9514858c71fb22c5ed93d651ea24bc6c05)
Reviewed-on: https://gerrit.libreoffice.org/19336
Tested-by: Michael Stahl <mstahl@redhat.com>
---
 sw/source/core/crsr/crossrefbookmark.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sw/source/core/crsr/crossrefbookmark.cxx b/sw/source/core/crsr/crossrefbookmark.cxx
index 9ec38bd..f54586a 100644
--- a/sw/source/core/crsr/crossrefbookmark.cxx
+++ b/sw/source/core/crsr/crossrefbookmark.cxx
@@ -84,7 +84,7 @@ namespace sw { namespace mark
         const vcl::KeyCode& rCode,
         const OUString& rName,
         const OUString& rShortName)
-        : CrossRefBookmark(rPaM, rCode, rName, rShortName, IDocumentMarkAccess::GetCrossRefHeadingBookmarkNamePrefix())
+        : CrossRefBookmark(rPaM, rCode, rName, rShortName, CrossRefNumItemBookmark_NamePrefix)
     { }
 
     bool CrossRefNumItemBookmark::IsLegalName(const OUString& rName)
-- 
2.4.3

